### ISSUE TITLE
PNR Tab UI | Bag Tag Table PNR Highlighting not Working

### ISSUE THREAD

**Issue by donleyac** [US-CBP/GTAS#706]

**Comment by donleyac** [comment]
@dje04001 

The highlighting feature basically checks the raw pnr for the TBD segment and Bag ID and highlights all those that match those properties. In the sample data provided this is how the Bag ID's appear in the database. 


![uploads/d228a054-e1c5-4f76-ba73-6b9ccf080d9f/image.png](https://s3-us-west-2.amazonaws.com/prod.huboard.com/uploads%2Fd228a054-e1c5-4f76-ba73-6b9ccf080d9f%2Fimage.png)

This is supposed to be the corresponding raw pnr section:


![uploads/09956d94-76a7-4c00-bc82-bd46df697733/image.png](https://s3-us-west-2.amazonaws.com/prod.huboard.com/uploads%2F09956d94-76a7-4c00-bc82-bd46df697733%2Fimage.png)


Basically we are looking for bag id 0001:1 however, the ID in the raw is 0001. Therefore I believe this is actually a parsing issue which is giving us the wrong id in the database.
